hpat/tests/test_series.py

```diff
@@ -1757,6 +1757,34 @@ def test_impl(S):
         S = pd.Series([np.nan, 2., 3.])
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
+    def test_series_notna_noidx(self):
+        def test_impl(S):
+            return S.notna()
+
+        hpat_func = hpat.jit(test_impl)
+
+        data_test = [[6, 6, 2, 1, 3, 3, 2, 1, 2],
+                     [1.1, 0.3, 2.1, 1, 3, 0.3, 2.1, 1.1, 2.2],
+                     [6, 6.1, 2.2, 1, 3, 0, 2.2, 1, 2],
+                     [6, 6, 2, 1, 3, np.inf, np.nan, np.nan, np.nan],
+                     [3., 5.3, np.nan, np.nan, np.inf, np.inf, 4.4, 3.7, 8.9]
+                     ]
```

Comment:
I propose to use globally defined data as input data in tests: https://github.com/IntelPython/hpat/commit/f2435b037ecc403c6adada060b94279c6ac2ca31#diff-deca39d332649cea819383154a5d2cb3R39-R62